fastplotlib/layouts/_utils.py

```diff
@@ -2,43 +2,12 @@
 
 import pygfx
 from pygfx import WgpuRenderer, Texture, Renderer
-from pygfx.renderers.wgpu.engine.renderer import (
-    EVENT_TYPE_MAP,
-    PointerEvent,
-    WheelEvent,
-)
 
-from wgpu.gui import WgpuCanvasBase
-
-from ..utils import gui
-
-
-# temporary until https://github.com/pygfx/pygfx/issues/495
-class WgpuRendererWithEventFilters(WgpuRenderer):
-    def __init__(self, target, *args, **kwargs):
-        super().__init__(target, *args, **kwargs)
-        self._event_filters = {}
-
-    def convert_event(self, event: dict):
-        event_type = event["event_type"]
-
-        if EVENT_TYPE_MAP[event_type] in [PointerEvent, WheelEvent]:
-            for filt in self.event_filters.values():
-                if (
-                    filt[0, 0] < event["x"] < filt[1, 0]
-                    and filt[0, 1] < event["y"] < filt[1, 1]
-                ):
-                    return
-
-        super().convert_event(event)
-
-    @property
-    def event_filters(self) -> dict:
-        return self._event_filters
```

Comment:
no longer required since events handled by imgui are no longer propagated to pygfx